packages/core/src/theme.js

```diff
@@ -0,0 +1,52 @@
+export const theme = {
```

Comment:
Theme is used both in core and legacy correct? are we having legacy import this theme from core? If not I'd probably recommend that since problems created by design drift I think would be noisier worse than problems caused by cross pollinating data between packages in this case

Reply:
Great idea, I just made legacy import from core.